src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { BrowserRouter, Routes, Route } from "react-router";
+import Nav from "./components/Nav";
+import HomePage from "./pages/HomePage";
+import FreeBoardPage from "./pages/FreeBoardPage";
+import ItemsPage from "./pages/ItemsPage";
+import AddItem from "./pages/AddItemPage";
+
+const App = () => {
+  return (
+    <BrowserRouter>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
```

Comment:
💊 제안
index 속성을 이용해 시작페이지라는 것을 더 명확하게 알 수 있게 하시는 것을 추천드립니다~

```suggestion
        <Route index element={<HomePage />} />
```